bindata/v3.11.0/openshift-apiserver/deploy.yaml

```diff
@@ -120,3 +121,13 @@ spec:
         node-role.kubernetes.io/master: ""
       tolerations:
       - operator: Exists
+      # Ensure that no more than one apiserver pod is deployed per master
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchLabels:
```

Comment:
on an upgrade, will this end up avoiding the existing daemonsets?

Reply:
Done. Updated deployment label to use `app: openshift-apiserver-a` to ensure deployment and daemonset won't try to manage the same pods.